cves/CVE-2018-14574.yml

```diff
@@ -49,7 +49,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  There is a vulnerability that causes Open redirects in the django middleware.
+  This vulnerability occurs when certain settings are enabled and the URL pattern
+  accepts paths ending in slash. This allows attackers to create maliciously crafted
+  URL that can lead to redirects to other sites.
```

Comment:
This should either say " a maliciously crafted URL" or "maliciously crafted URLs"